app/views/Board.coffee

```diff
@@ -45,17 +48,18 @@ class Board
         cell.setBorder @Colors.emptyCellBorder
         @empty_cells[row].push cell
 
-  createCells: (width, symbols) =>
+  createCells: (width) =>
     @cells = []
     for row in [0...@dimension]
       @cells.push []
       for col in [0...@dimension]
-        cell = new @Cell col, row, width, @two, this, @clickHandler, symbols[@toIdx @boardValues[row][col]]
+        cell = new @Cell col, row, width, @two, this, @clickHandler, @symbols[@toIdx @boardValues[row][col]]
         cell.setColor @Colors.cell
         cell.setBorder @Colors.cellBorder
         @cells[row].push cell
 
   deleteCells: (solution) ->
+    console.log 'delete cells', solution
```

Comment:
I think we'll remove all the console.logs and tests in a later PR but good catch